website/docs/r/storage_bucket.html.markdown

```diff
@@ -10,7 +10,7 @@ description: |-
 
 Creates a new bucket in Google cloud storage service (GCS).
 Once a bucket has been created, its location can't be changed.
-[ACLs](https://cloud.google.com/storage/docs/access-control/lists) can be applied using the `google_storage_bucket_acl` resource.
+[ACLs](https://cloud.google.com/storage/docs/access-control/lists) can be applied using the [`google_storage_bucket_acl` resource](./storage_bucket_acl.html).
```

Comment:
Hmm - it looks like we're kind of all over the place in how we format links to other places in the docs. I (admittedly pretty arbitrarily) favour the ones where we qualify it starting from `/docs/` such as the link to `google_compute_region_backend_service` in [`google_compute_backend_service`](https://github.com/terraform-providers/terraform-provider-google/blob/2.0.0/website/docs/r/compute_backend_service.html.markdown) (apologies - I don't think I can link to line numbers in markdown). Would it be much trouble to edit this change (and future PRs) to use links with that form?

Reply:
Thanks ! It will be easier to link to datasource from resource or to resource from datasource.